### DISCUSSION TITLE
Contribution to platform Windows

### DISCUSSION THREAD

**Discussion by kazatsuyu** [zed-industries/zed#7049] [Feature Requests]
Hello!

I am a Windows user and am eagerly awaiting Windows support. I understand that the community is currently focused on supporting Linux, but is there room to accept contributions to Windows?

I don't think I can make that big of a contribution, but I think I can address small incompatibilities (e.g. [unix dependencies of OsStr in sqlez](https://github.com/zed-industries/zed/blob/dd74643993b13de285b184d6ab26cf0a16c00162/crates/sqlez/src/bindable.rs#L325)) and reduce build errors.

If it is not too much of a burden on the development team to see such pull requests, I will work on it. If it is not the right time, I will wait for a roadmap for Windows support.

**Comment by mikayla-maki** [comment]
Feel free to make those PRs! We appreciate all the help we can get :)

**Reply by kazatsuyu** [reply]
Thanks!
